### PR TITLE
added new platform for the Unitymedia Horizon HD Recorder

### PR DESCRIPTION
**Pull request by benleb**
**Description:**
A new platform for the Unitymedia Horizon HD Recorder sold by Unitymedia in Germany.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#14275

## Checklist:

- [X] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [X] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

**Comment by frenck** [comment]
PR needs to be rebased in order to fix the build error.

**Comment by syssi** [comment]
Could you rebase the PR? Does the docs apply to the updated component?

**Comment by benleb** [comment]
done! :)

File: source/_components/media_player.horizon.markdown [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+    type: int
+  name:
+    description: The name of the device used in the frontend.
+    required: false
+    type: string
+{% endconfiguration %}
+
+#### {% linkable_title Preparation of the Horizon HD Recorder %}
+
+Makes sure to enable the Home Network ("Heimnetzwerk") service in the settings menu of the Horizon. Once you have set up the Media Library ("Medienbibliothek"), we can determine whether the device is turned on or off. Without this, the component will fail to start.
```
Comment (syssi) [inline source/_components/media_player.horizon.markdown]: The Home Networking ("Heimnetzwerk") service needs to be enabled at the settings menu of the receiver.

**Comment by benleb** [comment]
fixed ;) thanks!

**Review by syssi — APPROVED** [review]

**Pull request merged**